Active Documents/contenteditable-disabled.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset='utf-8'>
+    <title>ContentEditable-Disabled</title>
+    <script class='remove'>
+        var respecConfig = {
+            specStatus: "ED",
+            editors: [
+                {
+                    name: "Johannes Wilm",
+                    mailto: "johannes@fiduswriter.org",
+                    company: "Invited Expert"
+                },
+                {
+                    name: "Bo Cupp",
+                    mailto: "pcupp@microsoft.com",
+                    company: "Microsoft"
+                },
+                {
+                    name: "Grisha Lyukshin",
+                    mailto: "glyuk@microsoft.com",
+                    company: "Microsoft"
+                }
+            ],
+            github: "https://github.com/w3c/editing",
+            shortName: "contenteditable-interface",
+            wg: "Web Applications Working Group",
+            wgURI: "https://www.w3.org/2019/webapps/",
+            wgPatentURI: "https://www.w3.org/2004/01/pp-impl/114929/status",
+            license: "w3c-software-doc"
+        };
+    </script>
+    <script src='https://www.w3.org/Tools/respec/respec-w3c-common' class='remove'></script>
+</head>
+
+<body>
+    <section id='abstract'>
+        <h2>Abstract</h2>
+        <p>
+            This specification defines an API that indicates to user agents which editing-related commands are applicable to contenteditable regions of a document.  User agents SHOULD take this information into account when presenting any editing-related UI to ensure its applicability.
```

Comment:
Avoid RFC2119 keywords in non-normative sections. 
```suggestion
            This specification defines an API that indicates to user agents which editing-related commands are applicable to contenteditable regions of a document.  User agents can take this information into account when presenting any editing-related UI to ensure its applicability.
```

Reply:
The 'its' is referring to editing-related UI in that sentence.  The API provides hints as to what editing operations (and therefore what editing UI) are applicable in a given contenteditable element. We could change it to... "User agents can take this information into account when presenting any editing-related UI to ensure it is applicable to a given contenteditable region."

Reply:
I pushed another commit. It caused github to disable Commit Suggestion button. I'll make a change on my and push it here.